### PR TITLE
Mails

### PR DESCRIPTION
**Pull request by florianGierlichs**

### CODE REVIEW

**Comment by florianGierlichs** [comment]
Storybook:

https://company-esca-mails-vgz5gapphza.herokuapp.com/storybook/index.html?path=/story/mail--gelesen

File: client/src/components/Desktop/Email/MailCommunications.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import MailSenden from './MailSenden';
+import MailGesendet from './MailGesendet';
+import MailGelesen from './MailGelesen';
+import MailUngelesen from './MailUngelesen';
+
+export default function MailCommunications() {
+  const [activeNavigation] = useContext(MailNavigationContext);
+
+  if (activeNavigation === 'senden') {
+    return <MailSenden />;
```
Comment (Susanne1307) [inline client/src/components/Desktop/Email/MailCommunications.js:12]: I find the English/German mix a little strange, but I can't think of a better solution right now

File: client/src/components/Desktop/Email/MailContent.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem
+      ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod
+      tempor invidunt ut labore et dolore magna aliquyam erat, sed diam
+      voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet
+      clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit
+      amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, At accusam
+      aliquyam diam diam dolore dolores duo eirmod eos erat, et nonumy sed
+      tempor et et invidunt justo labore Stet clita ea et gubergren, kasd magna
+      no rebum. sanctus sea sed takimata ut vero voluptua. est Lorem ipsum dolor
+      sit amet. Lorem ipsum dolor sit amet, consetetur
```
Comment (Susanne1307) [inline client/src/components/Desktop/Email/MailContent.js:56]: Screenshot 2020-06-08 at 10 02 45

**Comment by Susanne1307** [comment]
Looks good :)

**Pull request merged**